src/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -215,6 +215,20 @@ private void filter(List<Integer> toShow, List <Integer> toHide) {
             toShow.add(R.id.action_send_file);
         }
 
+        // FAVORITES
+        if (mFile == null || downloading || uploading || mFile.isFolder() || mFile.keepInSync()) {
```

Comment:
Is this a good idea?
As in /core favorite files are something different. And maybe we want to support them in the future (and have them called "isStarred)? Then we would have different namings for same thing.